pages/wishes/index.vue

```diff
@@ -122,6 +122,15 @@ export default {
     },
     avatarColor(fullName) {
       return randomMC.getColor({ text: fullName })
+    },
+    wishDelete(wish){
+      this.deleteWish({
+        wish: wish
+      }).then(() =>{
+        this.getWishes()
+        }).catch(()=>{
+          this.throwAccessDenied()
```

Comment:
throwAccessDenied() is unnecessary at this point, because server can return another error.